server/src/test/java/org/spine3/server/SubscriptionServiceShould.java

```diff
@@ -0,0 +1,291 @@
+        subscriptionService.subscribe(topic, subscribeObserver);
+
+        // Activate subscription
+        final MemoizeStreamObserver<SubscriptionUpdate> activateSubscription = spy(new MemoizeStreamObserver<SubscriptionUpdate>());
+        subscriptionService.activate(subscribeObserver.streamFlowValue, activateSubscription);
+
+        // Cancel subscription
+        subscriptionService.cancel(subscribeObserver.streamFlowValue, new MemoizeStreamObserver<Response>());
+
+        // Post update to Stand
+        final ProjectId projectId = ProjectId.newBuilder().setId("some-other-id").build();
+        final Message projectState = Project.newBuilder().setId(projectId).build();
+        boundedContext.getStand().update(projectId, AnyPacker.pack(projectState));
+
+        // The update must not be handled by the observer
+        verify(activateSubscription, never()).onNext(any(SubscriptionUpdate.class));
+        verify(activateSubscription, never()).onCompleted();
+    }
+
+    private static BoundedContext newBoundedContext(String name) {
+        final Stand stand = Stand.newBuilder().setStorage(InMemoryStandStorage.newBuilder().build()).build();
+
+        return BoundedContext.newBuilder()
+                             .setStand(stand)
+                             .setName(name)
+                             .setStorageFactory(InMemoryStorageFactory.getInstance())
+                             .build();
+
+    }
+
+    private static BoundedContext setupBoundedContextForAggregateRepo() {
+        final Stand stand = Stand.newBuilder()
+                                 .setStorage(InMemoryStandStorage.newBuilder().build())
+                                 .setCallbackExecutor(MoreExecutors.directExecutor())
+                                 .build();
+
+        final BoundedContext boundedContext = BoundedContext.newBuilder()
+                                                            .setStand(stand)
+                                                            .setStorageFactory(InMemoryStorageFactory.getInstance())
+                                                            .build();
+
+        stand.registerTypeSupplier(new Given.ProjectAggregateRepository(boundedContext));
+
+        return boundedContext;
+    }
+
+    private static class MemoizeStreamObserver<T> implements StreamObserver<T> {
+
+        private T streamFlowValue;
+        private Throwable throwable;
+        private boolean isCompleted;
+
+        @Override
+        public void onNext(T value) {
+            this.streamFlowValue = value;
+        }
+
+        @Override
+        public void onError(Throwable t) {
+            this.throwable = t;
+        }
+
+        @Override
+        public void onCompleted() {
+            this.isCompleted = true;
+        }
+
+        private void checkFields() {
+            checkFields(true);
+        }
+
+        private void checkFields(boolean isCompleted) {
```

Comment:
This is `verifyState`.

`checkFields` may be confusing, since we also have `fieldMask` as a part of the domain language.